modules/navier_stokes/src/physics/WCNSFVFlowPhysics.C

```diff
@@ -1349,6 +1384,51 @@ WCNSFVFlowPhysics::getPorosityFunctorName(bool smoothed) const
     return _porosity_name;
 }
 
+MooseFunctorName
+WCNSFVFlowPhysics::getLinearFrictionCoefName() const
+{
+  // Check all blocks. If more than one block, they would need to be consolidated #include in
+  // a single functor material. We won't implement this for now
+  if (_friction_types.empty())
+    return "0";
```

Comment:
I would prefer returning an empty string unless that's going to cause issues for some reason

Reply:
Checking for an empty string is a common operation; checking for a one-character string that is "0" is less so

Reply:
done